src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -138,37 +140,30 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
                       size="xs"
                       direction={sort.asc ? 'up' : 'down'}
                     />
-                  </StyledSortLink>
-                </div>
-                <div>{t('Actions')}</div>
-              </TableLayout>
-            </PanelHeader>
-
-            {loading ? (
-              <LoadingIndicator />
-            ) : (
-              this.tryRenderEmpty() ?? (
-                <PanelBody>
-                  <Projects orgId={orgId} slugs={Array.from(allProjectsFromIncidents)}>
-                    {({initiallyLoaded, projects}) =>
-                      ruleList.map(rule => (
-                        <RuleListRow
-                          // Metric and issue alerts can have the same id
-                          key={`${isIssueAlert(rule) ? 'metric' : 'issue'}-${rule.id}`}
-                          projectsLoaded={initiallyLoaded}
-                          projects={projects as Project[]}
-                          rule={rule}
-                          orgId={orgId}
-                          onDelete={this.handleDeleteRule}
-                        />
-                      ))
-                    }
-                  </Projects>
-                </PanelBody>
-              )
-            )}
-          </Panel>
-
+                  </StyledSortLink>,
+                  t('Actions'),
+                ]}
+                isEmpty={false}
```

Comment:
`<PanelTable>` also handles this empty state too, instead of hard coding it to false, we can take the logic from `tryRenderEmpty` and use it for this prop, and `emptyMessage`.

Reply:
updated